portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -74,7 +74,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
           comment = translation.getTranslatedText();
         }
       
-        Comment commentObject = new Comment(id, name, comment, timestamp);
+        Comment commentObject = new Comment(id, name, comment, timestamp, score);
```

Comment:
I had to make this change because when resolving merge conflicts I accidentally removed score from the parameters.